web/text_layer_builder.css

```diff
@@ -48,10 +48,12 @@
 
 /* Only necessary in Google Chrome, see issue 14205, and most unfortunately
```

Comment:
Nit: you could skip the comment lines too.

Reply:
I would, but unfortunately that doesn't work :-(

This is using the old pre-processor, which doesn't handle comments all that well. Rather than trying to fix that, it might be easier to just remove all comments from CSS files during building.